Composer/packages/extensions/visual-designer/src/schema/uischema.tsx

```diff
@@ -152,6 +152,27 @@ export const uiSchema: UISchema = {
         </>
       ) : null,
   },
+  [SDKTypes.SkillDialog]: {
+    'ui:widget': CardTemplate,
+    header: {
+      'ui:widget': ActionHeader,
+    },
+    body: {
+      'ui:widget': DialogRef,
+      dialog: data => data.dialog,
+      getRefContent: data => dialogRef => (
+        <>
+          {dialogRef || '?'} <FixedInfo>(Dialog)</FixedInfo>
+        </>
+      ),
+    },
```

Comment:
@yeze322 update the content displayed here

Reply:
Ah, yeah I haven't cleaned up the node renderer yet!

Reply:
It's updated